### ISSUE TITLE
Error 2128 on rebuild

### ISSUE THREAD

**Issue by HughWarrington** [joyfullservice/msaccess-vcs-addin#552]
I'm using addin version 4.0.34, Office 365 64-bit, Win10 64-bit.

I did a full export of my frontend .mdb, having set both the sanitise options to Off (I had to manually apply the fix for #537 first). Then took the .src folder and ran a Build As....

During the rebuild I got an error dialog:

```
Run-time error '2128':

Microsoft Access encountered errors while importing BackendCreate.
```

Here's some relevant info from the debugger at the line of code with the exception:

![image](https://github.com/user-attachments/assets/334aa90f-167e-4b67-9a97-a760433d2e1d)

I'm afraid I can't upload the whole BackendCreate.bas file for commercial reasons. But it is notable that by alphabetical order, it is the first file inside the forms folder. In other words, it was probably the first form to be imported, suggesting a systematic error affecting all forms rather than just this one.

I didn't get this error until I changed the sanitise options to Off. Previously they were both set to Standard.

**Comment by hecon5** [comment]
I have also had this happen...and it turned out to be a reference in that file which didn't correctly load. 

@joyfullservice: if you turn off Sanitize options, does the UTF8 conversion still happen? I thought it did, but now I'm not so sure.

**Comment by joyfullservice** [comment]
> 
> @joyfullservice: if you turn off Sanitize options, does the UTF8 conversion still happen? I thought it did, but now I'm not so sure.

Yes, the export file is written in UTF-8 regardless of the sanitize option used. Here is the line in `SaveComponentAsText` that writes the file as UTF-8. (The default encoding for the `WriteFile` function.)

![image](https://github.com/user-attachments/assets/95cd7b0a-c977-492f-94fc-27977e606ce4)

**Comment by HughWarrington** [comment]
More info. I've just found a file called errors.txt in the build directory:

```
Microsoft Access encountered an error while importing the object 'BackendCreate'.

Error encountered at line 39.
This property does not apply to this control.
```

Here is that line in BackendCreate.bas:
![image](https://github.com/user-attachments/assets/fe06f05a-9382-405c-a263-74717a3d357b)

**Comment by HughWarrington** [comment]
> Are you able to perform an export and build with Sanitize level set to `Standard`?

Yes.

I only noticed the problem when I switched both levels to Off. I didn't try the Minimal or Extended levels.

This isn't a blocker for me. I only turned sanitisation off to see if it would help with #551.

**Comment by HughWarrington** [comment]
Sorry I probably should have said sooner! The problem is specifically when sanitise is Off.

**Comment by joyfullservice** [comment]
No worries. I think removing the Sanitize -> None option should help other users avoid similar issues in the future.

**Comment by hecon5** [comment]
@HughWarrington  @joyfullservice since `eslNone` is now disabled, can this be closed?
